Example/ViewController.swift

```diff
@@ -54,20 +54,44 @@ class ViewController: UIViewController {
             }
         }
     }
+    
+    var currentRouteIndex = 0 {
```

Comment:
I intentionally did not use `IndexedRouteResponse` to store this as it is convenient to have custom `didSet` for index update.